.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: python
 
+os: linux
```

Comment:
All of the existing jobs explicitly specified `os`, so I think the configs are already clear.
Did you think it has ambiguous parts?